### DISCUSSION TITLE
How to access organisation members github account email

### DISCUSSION THREAD

**Discussion by atifpropelius** [community/community#119650] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

I have all the scope , and have access token through oauth of the admin of the organisation in a github . 

I want to get all the members email address , but right now i am getting empty email 

GET https://api.github.com/users/USERNAME

response : -
{
    "login": "gentleatif",
    "id": 69183987,
    "node_id": "MDQ6VXNlcjY5MTgzOTg3",
    "avatar_url": "https://avatars.githubusercontent.com/u/69183987?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/gentleatif",
    "html_url": "https://github.com/gentleatif",
    "followers_url": "https://api.github.com/users/gentleatif/followers",
    "following_url": "https://api.github.com/users/gentleatif/following{/other_user}",
    "gists_url": "https://api.github.com/users/gentleatif/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/gentleatif/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/gentleatif/subscriptions",
    "organizations_url": "https://api.github.com/users/gentleatif/orgs",
    "repos_url": "https://api.github.com/users/gentleatif/repos",
    "events_url": "https://api.github.com/users/gentleatif/events{/privacy}",
    "received_events_url": "https://api.github.com/users/gentleatif/received_events",
    "type": "User",
    "site_admin": false,
    "name": "Atif Hussain",
    "company": null,
    "blog": "",
    "location": null,
    "email": null,
    "hireable": null,
    "bio": "Simple Living Higher Thinking",
    "twitter_username": null,
    "public_repos": 65,
    "public_gists": 0,
    "followers": 1,
    "following": 4,
    "created_at": "2020-08-04T04:56:22Z",
    "updated_at": "2024-04-13T14:15:29Z"
}

So i am getting email over here is there any way i can get all the members orgs email .

**Comment by atifpropelius** [comment]
facing same issue here

**Comment by SMillerDev** [comment]
Were you using finegrained access tokens? Because I am experiencing the same issue when I use those.
